### ISSUE TITLE
system-wide install instructions should be changed 

### ISSUE THREAD

**Issue by manfer** [nodejs/node-v0.x-archive#25500]
It is not a good idea to mess up system folders with unnecesary files. The LICENSE, README.md and ChangeLog files has to be excluded when extracting. So it would be a good idea to change the system-wide instructions provided in README.md.

```
cd /usr/local && tar --strip-components 1 -xzf \
                    /path/to/node-<version>-<platform>-<arch>.tar.gz \
                    --exclude='node-*/ChangeLog' \
                    --exclude='node-*/LICENSE' \
                    --exclude='node-*/README.md'
```

**Comment by sam-github** [comment]
Its surprisingly difficult to figure out how to install the linux tarballs on a system, I've regularly come into contact with customers who aren't installing node very well, the docs could definitely use some help on this. @manfer can you PR the README, and be prepared for some discussion?

/cc @rmg

**Comment by rmg** [comment]
Ya, it's kind of unfortunate that there isn't a shorter way to do it.. As a result, I don't usually bother filtering them out since they happen to be harmless in _this_ case.

**Comment by manfer** [comment]
@rmg even if it is harmless it is not a good idea. If all packages do the same, systems will finish with a lot of files in unwanted places. As you comment it is unfortunate not having a shorter way.

Another way would be having different builds for local install or system-wide install. On system-wide install if those files needs to be installed should go into one of the doc folders the same as it is done when building and installing with *make install*. Probably not a good solution.

@sam-github sorry, I can't understand "PR the README" meaning.

It is difficult to decide how to install node in linux. Personally I don't like any method that doesn't make use of the package manager. So that excludes building the package and installing with *make install*, and excludes installing from tarball binary too. Besides, directly unpacking the tarball binary to */usr/local* with the command above, and if I'm not wrong, we install the dev package too which is not necessary for most node users.

So, I was going to install from binary tarball when I opened this issue but finally I decided to create rpms myself. Something that was easy because there is a github project [nodejs-rpm](https://github.com/kazuhisya/nodejs-rpm) with the spec file needed to create rpms from node source which is up to date.

For distros based on enterprise linux there are repositories with versions of node but those are older versions than the current node version. The versions I found in epel and nodesource were 0.10.x. Don't recall the exact build number but it starts with 0.10.
